cypress/e2e/list-management/related-links.feature

```diff
@@ -15,9 +15,9 @@ Feature: Related links
     And I click the link "Add link"
     And I enter "Where to find eggs" for "Page title"
     And I enter "https://gov.uk" for "URL"
-    And I click the link "Continue"
+    And I continue
```

Comment:
The step definition for `I click the link "Continue"` wasn't clicking on a link specifically. It was just looking for text and clicking it. 


previously:
```
When("I click the link {string}", (string) => {
  cy.findAllByText(string).eq(0).click();
});
```

I changed it, so "I click the link" is now only searching for links, but that ended up breaking a bunch of tests because the "links" were actually buttons. 

```
When("I click the link {string}", (string) => {
  cy.findAllByRole("link", {
    name: string,
  }).click();
});
```

`I continue` is looking for specifically a button.

Reply:
Ah I see. Would `I click continue` make more sense instead of `I continue`?

Reply:
It would make more sense in terms of how it's been implemented, but "clicking" is not accessible language. I'd like to change them eventually.. how realistic that is 🤷. 

The action is "continue" - how the user does that (could be click, tap, press, voice over) we shouldn't "care" about. Yet to figure out the best way to do that instead of .click(), though 🙂